src/test/java/seleniumeasyPage/AlertPage.java

```diff
@@ -12,32 +14,39 @@
 
 public class AlertPage {
     private final WebDriver driver;
-    private final By ALERT_BUTTON = By.xpath("//button[contains(@onclick,'Alert')]");
-    private final By CONFIRM_BUTTON = By.xpath("//button[contains(@onclick,'Confirm')]");
-    private final By PROMPT_BUTTON = By.xpath("//button[contains(@onclick,'Prompt')]");
-    private final By CONFIRM_BOX_TEXT = By.id("confirm-demo");
-    private final By PROMPT_BOX_TEXT = By.id("prompt-demo");
+    @FindBy(xpath = "//button[contains(@onclick,'Alert')]")
+    WebElement ALERT_BUTTON;
```

Comment:
Please fix variables naming. It should be in camel case instead of upper case

Reply:
fixed